.github/workflows/php.yml

```diff
@@ -28,12 +28,17 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install libmemcached
+      run: sudo apt-get install -y libmemcached-dev
+
     - name: Install PHP with extensions
       uses: shivammathur/setup-php@v2
       with:
         php-version: ${{ matrix.php-version }}
-        extensions: memcached
+        extensions: memcached-3.2.0
         coverage: none
+      env:
+        fail-fast: true
```

Comment:
@iFixit/qae We should always set this:

>By default, expect composer tools which cannot be set up gracefully leave an error message in the logs, the execution is not interrupted. To change this behaviour you can set fail-fast flag to true.

https://github.com/shivammathur/setup-php

Reply:
That's rough that it's not automatically set.